osidb/serializer.py

```diff
@@ -1659,6 +1659,13 @@ def update(self, new_flaw, validated_data):
         if old_flaw.is_embargoed and self._is_public(new_flaw, validated_data):
             new_flaw.unembargo()
 
+        # Force Jira task creation if requested
+        request = self.context.get("request")
+        if request:
+            create_jira_task = request.query_params.get("create_jira_task")
```

Comment:
This parameter should be added to OpenAPI schema via `@extend_schema_view` decorator in [Flaw API view](https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/api_views.py#L419)